tests/config/templates/condition.yaml

```diff
@@ -0,0 +1,86 @@
+{{- define "condition" }}
+  {{- $root := index . 0 }}
+  {{- $condition := index . 1 }}
+  {{- $expect := index . 2 }}
+  {{- if eq (include "_config.condition" (list $root $condition)) $expect }}
+    {{- printf "# Condition %q is %q" $condition $expect }}
```

Comment:
Using the actual value here instead of `$expect` is probably better, in case encounter cases where two things compare equal but are not actually equal (though I think the result is either `"true"` or `"false"` so I don't think that can happen… but weird things is what tests are for right?).

Reply:
Given that we just did a string comparison, I don't think this is actually possible, except for the special `"<no value>"` string, that prints as an empty string but compares as its actual value.

But since we need to store the actual value for the failure case anyways, I'll change it here too.

Reply:
Fixed in d120de6